WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,9 +1,6 @@
-Welcome to version 21! In honor of the occasion, we added extensions so you can save content as a draft and share content to Jetpack using your device’s built-in menus. You’ll also get more detailed push notifications on your device.
+* [**] We added a new landing screen with a cool animation that responds to device motion! [#19251, #19264, #19277, #19381, #19404, #19410, #19432, #19434, #19442, #19443, #19468, #19469]
+* [*] [internal] Database access change: the 'new Core Data context structure' feature flag is turned on by default. [#19433]
+* [***] Widgets are now on Jetpack. Find Today, This Week, and All Time Widgets to display your Stats on your home screen. [#19479]
+* [*] Block Editor: Fixed iOS Voice Control support within Image block captions. [https://github.com/WordPress/gutenberg/pull/44850]
+* [***] Dropped support for iOS 13. Now supporting iOS 14.0 and above. [#19509]
```

Comment:
In hindsight, it would have been good to add a line in the release notes for the previous version (21.0) saying that it would be the last one supporting iOS 13.

We established that's something that affects only a minor portion of users, but I still think it would be nice to it mentioned in the App Store, if someone on an old device wonders why their app hasn't been updated in a while.

> **Heads up: This will be the last version to support iOS 13. Make sure to update to a newer version of iOS to continue receiving Jetpack updates.
>
> Other release notes...

Reply:
Can we retrospectively update the Release Notes of a previous version in ASC? 🤔  Especially for those cases?

I would be surprised if we can edit release notes for versions other than the latest, but maybe worth double-checking anyway?